scripts/main_hcs.py

```diff
@@ -13,16 +13,20 @@
 from qtpy.QtWidgets import *
 from qtpy.QtGui import *
 
+# Add the parent directory of squid_control to sys.path
+script_dir = os.path.dirname(os.path.abspath(__file__))
+parent_dir = os.path.abspath(os.path.join(script_dir, '..'))
+sys.path.insert(0, parent_dir)
+
 # app specific libraries
 import squid_control.control.gui_hcs as gui
 
-from configparser import ConfigParser
+from configparser import ConfigParsers
```

Comment:
This should be `ConfigParsers`
```suggestion
from configparser import ConfigParser
```